Uplift/Views/GymDetail/GymDetailFacilitiesCell.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  GymDetailFacilitiesCell.swift
+//  Fitness
+//
+//  Created by Yana Sang on 9/5/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class GymDetailFacilitiesCell: UICollectionViewCell {
+
+    // MARK: - Constraint constants
+    enum Constants {
+        static let dividerHeight = 1
+        static let dividerTopPadding = 24
+        static let gymFacilitiesCellHeight: CGFloat = 20
+        static let gymFacilitiesTopPadding = 12
+        static let facilitiesLabelHeight = 22
+        static let facilitiesLabelTopPadding = 23
+    }
+
+    // MARK: - Public data vars
+    static var baseHeight: CGFloat {
+        return CGFloat(Constants.facilitiesLabelTopPadding + Constants.facilitiesLabelHeight + Constants.gymFacilitiesTopPadding + Constants.dividerTopPadding + Constants.dividerHeight)
+    }
+
+    // MARK: - Private view vars
+    private let dividerView = UIView()
+    private let facilitiesLabel = UILabel()
+    private var gymFacilitiesTableView: UITableView!
+    static var gymFacilitiesCount: Int = 0
+
+    // MARK: - Private data vars
+    private let facilitiesData: [String: [String]] = [
+        "7c53229a64f4794f57a715a9ec0c7f806db23514": ["Fitness Center"],
```

Comment:
We use these ID's in `OldGemDetailViewController` too so we should move them into a global Constants file